packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -40,21 +40,32 @@ export class Checkboxes extends GOVUKFrontendComponent {
     /** @satisfies {NodeListOf<HTMLInputElement>} */
     const $inputs = $module.querySelectorAll('input[type="checkbox"]')
     if (!$inputs.length) {
-      return this
+      throw new ElementError(null, {
+        componentName: 'Checkboxes',
+        identifier: '<input type="checkbox">'
```

Comment:
No problem with this at all, I think it's much friendlier

But now all the other JavaScript variable `identifier: '$module'` options jump out at me

Thinking ahead then, wonder if we should stick to CSS selectors?

### Before
```mjs
// Checkboxes: $module not found
identifier: `$module`
```

```mjs
// Checkboxes: <input type="checkbox"> not found
identifier: `<input type="checkbox">`
```

### After
```mjs
// Checkboxes: [data-module="govuk-checkboxes"] not found
identifier: `[data-module="${Checkboxes.moduleName}"]`
```

```mjs
// Checkboxes: input[type="checkbox"] not found
identifier: 'input[type="checkbox"]'
```